### PR TITLE
Front-end revamp of slot-booking webpage 

### LINKED ISSUE #8: Revamp the front-end of mappy.html
**Is your feature request related to a problem? Please describe.**
The front-end design of `mappy.html` looks very clunky and hard to use.

**Describe the solution you'd like**
A complete redoing of the front-end part of `mappy.html`.

**Describe alternatives you've considered**

**Additional context**
Screenshot:
Screen Shot 2021-09-16 at 8 07 37 PM

### PR DESCRIPTION
**Pull request by prateekrao**
#### Issue Number
fixes #8 

#### Describe the changes you've made
Changed `mappy.html` front-end completely, giving it a newer and more minimalistic look.
Added slot options and created button for uploading the files.
Changed the css file appropriately.

#### Describe if there is any unusual behaviour of your code ( Write `NA` if there isn't )
`NA`

#### Test plan ( Optional but Recommended )
A good test plan should give instructions that someone else can easily follow.
How can someone test your code?

#### Checklist

- [x] My code follows the code style of this project.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: PRNTRS Client/mappy.html [hunk 1]
```diff
@@ -1,20 +1,52 @@
...
+            <option value="early">7.00AM</option>
+            <option value="morning">9.30AM</option>
+            <option value="noon">12.00PM</option>
+            <option value="afternoon">3.00PM</option>
+            <option value="evening">5.00PM</option>
+            <option value="twilight">8.00PM</option>
+            <option value="night">10.00PM</option>
+        </select>
+
+        <button type="submit" name="Confirm Slot Booking">Confirm</button>
```
Comment (vedanthv) [inline PRNTRS Client/mappy.html:46]: @prateekrao Hey Prateek, I am going to implement Payment Gateway Checkout for the client. Is this button meant to serve that purpose?

**Review by vedanthv — APPROVED** [review]

File: PRNTRS Client/mappy.html (hunk 1 above)
Comment (prateekrao) [reply PRNTRS Client/mappy.html:46]: Yes @vedanthv that was the idea :)

Comment (vedanthv) [reply PRNTRS Client/mappy.html:46]: > Yes @vedanthv that was the idea :)

Cool! Thanks. I have closed the review

**Comment by A-HK** [comment]
Hey @prateekrao , please add a screenshot of the page ?

**Comment by prateekrao** [comment]
@A-HK 
Have attached the screenshot.
Screen Shot 2021-09-22 at 11 13 54 AM

**Comment by A-HK** [comment]
Ahh I see. So you haven't applied any CSS for the upload file button and the confirm button, correct?
@prateekrao 

Is it possible to take some time out to include the CSS styling in this PR or would you prefer to move on to an ML project now? XD

**Comment by prateekrao** [comment]
I'll work on it xD, shouldn't take much time.

**Comment by vedanthv** [comment]
> Ahh I see. So you haven't applied any CSS for the upload file button and the confirm button, correct?
> @prateekrao
> 
> Is it possible to take some time out to include the CSS styling in this PR or would you prefer to move on to an ML project now? XD

@A-HK Can you merge this PR and open another one for CSS. I would like to work on the Payment Gateway :)

**Comment by A-HK** [comment]
Okay cool. Thanks for your contribution @prateekrao🎉, feel free to open another PR just for the CSS as soon as you're done :)

**Pull request merged**